compute_tools/src/compute.rs

```diff
@@ -1559,6 +1571,21 @@ impl ComputeNode {
             });
         }
 
+        // Configure rsyslog for HIPAA audit logging
+        if let ComputeAudit::Hipaa = pspec.spec.audit_log_level {
+            let remote_endpoint = std::env::var("AUDIT_LOGGING_ENDPOINT").unwrap_or("".to_string());
+            if remote_endpoint.is_empty() {
+                anyhow::bail!("AUDIT_LOGGING_ENDPOINT is empty");
```

Comment:
NIT: not a rust expert, but instead of unwrapping with default to empty string you can probably check the Result of var() with match? It will be more clean

I assume that nobody explicitly set this war to the empty string outside of VM

Reply:
>I assume that nobody explicitly set this war to the empty string outside of VM

we set it to empty string in the [cloud config](https://github.com/neondatabase/cloud/blob/0e6615bf0f104d7e86577279843830faa1fc384c/helm/helmfile/control-plane/prod/prod-ap-southeast-1-epsilon.yaml#L57):


I guess I could just omit this field in spec and rely on default (not sure what is the default in this case, I guess `nil`?), but I wanted to be clear that this value is explicitly left empty, not just forgotten.